src/components/App.css

```diff
@@ -0,0 +1,22 @@
+
+.app {
+  background-color: #0E0E0E;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  letter-spacing: 2px;
+} 
+
+h1 {
+  font-family: 'Edo-sz';
+}
+
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.landing-page {
+  margin-block-start: 2em;
+  margin-block-end: 2em;
```

Comment:
Perhaps look at `margin` css property? `margin-block` is particularly for when you'er using some odd non right to left writing mode and has less browser support.